tests/framework/ensembleModelTests/test_ensemble_model_2_codes_optional_output.xml

```diff
@@ -6,14 +6,15 @@
     <created>2017-05-17</created>
     <classesTested>Models.Code, JobHandler.Thread</classesTested>
     <description>
-       The main goal of this test is to show how to specify optional output in the Ensemle Model object.
+       The main goal of this test is to show how to specify optional output in the Ensemble Model object.
+       All the RAVEN available data storage types are tested (DataObjects and Databases).
     </description>
```

Comment:
Hmm, the trouble I foresee with this is that the git log may not highlight only the change to the test file, or it may not mention the change to the input file at all, which could cause some confusion. If this is used by the regression test documentation then maybe it does make sense for developers to explicitly handle it. The downside of the current method is that there is no way to guarantee that these edits do not get forgotten. Maybe we could have a git hook that detects changes to xml files and asks a developer to write a brief description at commit time for the specific test file(s) edited? I would definitely classify this as a low-priority task though.